RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -376,5 +408,67 @@ private extension String {
     func isPromotionalLifetime(store: Store) -> Bool {
         return self.hasSuffix("_lifetime") && store == .promotional
     }
+}
+
+extension PurchaseInformation {
+
+    func billingInformation(localizations: CustomerCenterConfigData.Localization) -> String {
+        guard let expirationDate else {
+            // non subscription
+            return localizations[.pricePaid]
+                .replacingOccurrences(of: "{{ price }}", with: price.billingInformation(localizations: localizations))
+        }
 
+        if let introductoryDiscount {
+            if isCancelled {
+                var renewString = "\(introductoryDiscount.localizedPricePerPeriodByPaymentMode(.current))."
```

Comment:
Hmmm I am a bit worried about joining these two strings programmatically. I don't think it works for rtl languages. I think it should either be two different strings, or a whole string altogether in the backend

Reply:
I'll apply this in a new pr